src/base.jl

```diff
@@ -650,7 +645,7 @@ Return a TimeSeries.TimeArray where the forecast data has been multiplied by the
 component field.
 """
 function get_forecast_values(component::Component, forecast::Forecast)
-    return IS.get_forecast_values(component, forecast)
+    return IS.get_forecast_values(PowerSystems, component, forecast)
```

Comment:
What is the variable PowerSystems?

Reply:
In order to acquire the function represented by the label (which is a string) we have to call getfield on the module containing the function.  In this case the function resides in a different module than the get_forecast_values function, so we have to pass in the module name (PowerSystems).